cmake/syslog-ng-config.h.in

```diff
@@ -106,3 +106,4 @@
 #cmakedefine01 SYSLOG_NG_HAVE_SO_MEMINFO
 #cmakedefine01 SYSLOG_NG_ENABLE_AFSOCKET_MEMINFO_METRICS
 #cmakedefine01 SYSLOG_NG_HAVE_IV_WORK_POOL_SUBMIT_CONTINUATION
+#cmakedefine01 SYSLOG_NG_ENABLE_LIBUNWIND
```

Comment:
This needs to be set somewhere based on libunwind check results.

Reply:
Meh, I think we let this CMake idea go a long time ago (in production). It is extremely useful during development as it reduces compilation and configuration time to a few seconds. If we used it the modern way, it would also be much easier to use it.

TLDR: I'm merging this for v4.10.0. Let's fix the CMake part separately when we have time.